data-obfuscator/go.mod

```diff
@@ -0,0 +1,24 @@
+module github.com/helios/go-sdk/dataobfuscator
+
+go 1.18
+
+require (
+	github.com/ohler55/ojg v1.17.4
+	go.opentelemetry.io/otel v1.12.0
```

Comment:
Why does the obfuscator rely on otel?

Reply:
it returns an attribute

Reply:
Which?

Reply:
`func obfuscateAttributeValue(attribute attribute.KeyValue) attribute.Value {
	obfuscationConfig := getObfuscationConfig()
	hMacKey = []byte(obfuscationConfig.obfuscationhmacKey)
	if obfuscationConfig.obfuscationEnabled && slices.Contains(DATA_TO_OBFUSCATE, string(attribute.Key)) {
		return obfuscateDataHelper(attribute.Value, obfuscationConfig.obfuscationMode, obfuscationConfig.obfuscationRules)
	}
	return attribute.Value
}`

Reply:
👍